study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/controller/NotificationController.java

```diff
@@ -363,7 +363,7 @@ public ModelAndView viewNotificationList(HttpServletRequest request) {
             && notification
                 .getNotificationScheduleType()
                 .equals(FdahpStudyDesignerConstants.NOTIFICATION_NOTIMMEDIATE)) {
-          notification.setCheckNotificationSendingStatus("Not sent");
```

Comment:
Please update file copyright to add Google LLC and dated `2020-2021`

Reply:
Fixed PR comment